index.html

```diff
@@ -18,246 +18,311 @@
     <header>
       <nav id="header-container">
         <a href="#" class="header">My Logo</a>
-        <img src="./images/hamburger.svg" alt="menu-icon" class="menu" />
-      </nav>
-    </header>
-    <main>
-      <section class="hero-section">
-        <h1>I’m Belal Gamal Glad to see you!</h1>
-        <p>
-          I’m a software developer! I can help you build a product , feature or
-          website Look through some of my work and experience! If you like what
-          you see and have a project you need coded, don’t hestiate to contact
-          me.
-        </p>
-        <div class="connect-text">
-          <a href="#">LET’S CONNECT</a>
-        </div>
-
-        <div class="social-media">
+        <div class="nav-items">
           <ul>
             <li>
-              <a
-                href="https://twitter.com/belalgamal2016"
-                target="_blank"
-                rel="noopener"
-              >
-                <img src="./images/twitter.svg" alt="twitter-icon" />
-              </a>
-            </li>
-            <li>
-              <a
-                href="https://www.linkedin.com/in/belal-gamal-79b8a2133/"
-                target="_blank"
-                rel="noopener"
-              >
-                <img src="./images/linkedin.svg" alt="linkedin-icon" />
-              </a>
-            </li>
-            <li>
-              <a href="#" target="_blank" rel="noopener">
-                <img src="./images/Vector.svg" alt="vector-icon" />
-              </a>
+              <a href="#">Portoflio</a>
             </li>
             <li>
-              <a
-                href="https://github.com/belalgamal46"
-                target="_blank"
-                rel="noopener"
-              >
-                <img src="./images/github.svg" alt="github-icon" />
-              </a>
+              <a href="#"> About </a>
             </li>
             <li>
-              <a href="#" target="_blank">
-                <img src="./images/handVector.svg" alt="hand-vector-icon" />
-              </a>
+              <a href="#"> Contact </a>
             </li>
           </ul>
         </div>
```

Comment:
- [OPTIONAL] Kindly make sure that your links are working and redirect the user to the different sections of your project. :+1: :smiley: :smiley: